entries/f/flyfrontier.com.json

```
@@ -8,6 +8,13 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "do",
+      "sv",
+      "gt",
+      "mx",
+      "us"
```

Current list according to [Wikipedia](https://en.wikipedia.org/wiki/List_of_Frontier_Airlines_destinations):
```suggestion
      "ag",
      "bs",
      "bz",
      "cr",
      "cu",
      "do",
      "gt",
      "jm",
      "mx",
      "nl",
      "pr",
      "sv",
      "us"
```